spec/models/ability_spec.rb

```diff
@@ -71,6 +71,22 @@
     end
   end
 
+  %i[tutor organizer].each do |role|
+    it "can view and add notes to application letters" do
```

Comment:
This doesn't seem to be recommended as it was not used in the other tests above. (also "{role}" is not that meaningful, is it?)